{{cookiecutter.project_slug}}/home/api/v1/permissions.py

```diff
@@ -0,0 +1,22 @@
+import os
+import hmac
+
+from rest_framework.permissions import BasePermission
+
+
+class CrowboticsExclusive(BasePermission):
+    def has_permission(self, request, view):
+        header_signature = request.META.get("HTTP_CB_SIGNATURE", None)
+        secret = os.environ.get("CROWDBOTICS_SECRET", None)
```

Comment:
This could return false here if it's not set.

Reply:
I made only one single check instead of a check per item.